scripts/generate_nwb.py

```diff
@@ -6,6 +7,9 @@
 PWD = os.path.dirname(os.path.abspath(__file__))
 USER_HOME = os.path.expanduser("~")
```

Comment:
Is this used?

Reply:
If you are referring to PWD and USER_HOME, these are used to set the output and metadata paths below. Could be improved by setting up better central paths in later updates.